#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_include_PAGESHIFT_definition - Mattia Dongili <malattia@debian.org>
## as per file name
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian's standard location for uml_net helper

@DPATCH@

# Author: malattia
# Status: none

--- linux-source-2.6.18/arch/um/os-Linux/skas/process.c	2006-09-20 05:42:06.000000000 +0200
+++ linux-source-2.6.18/arch/um/os-Linux/skas/process.c.clean	2006-11-04 20:20:41.764186563 +0100
@@ -16,6 +16,7 @@
 #include <sys/user.h>
 #include <sys/time.h>
 #include <asm/unistd.h>
+#include <asm/page.h>
 #include <asm/types.h>
 #include "user.h"
 #include "sysdep/ptrace.h"
